### PR TITLE
introduce GitHub Action for CRON execution to detect regressions

### PR DESCRIPTION
**Pull request by agourlay**

### CODE REVIEW

File: .github/workflows/run-coach.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                  "type": "section",
+                  "text": {
+                    "type": "mrkdwn",
+                    "text": "Something is wrong.\nView the results <${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}|here>"
+                  }
+                }
+              ]
+            }
+        env:
+          SLACK_WEBHOOK_URL: ${{ secrets.COACH_CHANNEL_WEBHOOK_URL }}
```
Comment (agourlay) [inline .github/workflows/run-coach.yml:72]: this one needs additional setup

Comment (tellet-q) [reply .github/workflows/run-coach.yml:72]: I created a new webhook, ping me to get an url

File: coach-things.sh [hunk 2]
```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+
+set -e
+
+RUN_TIME=${1:-300}
+QDRANT_VERSION=${2:dev}
```
Comment (tellet-q) [inline coach-things.sh]: Shouldn't here be a hyphen? `${2:-dev}`

**Review by tellet-q — CHANGES_REQUESTED** [review]

File: coach-things.sh (hunk 2 above)
Comment (agourlay) [reply coach-things.sh]: Fixed, good for approval :)

**Review by tellet-q — APPROVED** [review]
lgtm

**Pull request merged**